corehq/apps/hqadmin/tests/test_authenticate_as.py

```diff
@@ -98,7 +98,7 @@ def test_authenticate_as(self):
 
         resp = self.client.post(reverse(AuthenticateAs.urlname), form.data, follow=True)
         self.assertEqual(
-            self.client.session['_auth_user_id'],
+            int(self.client.session['_auth_user_id']),
```

Comment:
@benrudolph Is this acceptable?  Just want to make sure I'm not hiding an actual error.